scripts/pull-down-unified-test-api-spec.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+TEST_API_DIR=$(realpath ./internal/api/test-api)
+
+# Data source
+API_SPEC_PATH=$TEST_API_DIR/unified-test-repo
+API_SPEC_BRANCH=${API_SPEC_BRANCH:-main}
+
+# Outputs
+TEST_API_GENERATED=$TEST_API_DIR/unified-test-spec
```

Comment:
Using the API version date here would be a nice way to indicate at-a-glance which version of the test API we're targeting. I think the parent directory `testapi` gives enough context.

```suggestion
TEST_API_GENERATED=$TEST_API_DIR/2024-10-15
```